lib/delocalize/parsers/date_time.rb

```diff
@@ -51,7 +51,7 @@ def default_parse(datetime, type)
 
         today = Date.current
         parsed = Date._parse(datetime)
-        return if parsed.empty? # the datetime value is invalid
+        raise ArgumentError, 'invalid date' if parsed.empty? # the datetime value is invalid
```

Comment:
I'm fine with doing this depending on your thoughts regarding the ArgumentError situation as discussed below.